packages/api-browser/window.js

```diff
@@ -2,10 +2,20 @@ if (!window.ssf) {
   window.ssf = {};
 }
 
+window.accessableWindows = [];
+
 class Window {
   constructor(url, name, features) {
-    window.open(url, name, objectToFeaturesString(features));
+    const win = window.open(url, name, objectToFeaturesString(features));
+    window.accessableWindows.push({
```

Comment:
What happens if the window closes? Feels like it should be removed from here, as it may lead to a memory leak and it would no longer be accessible. Same applies for adding the parent window, below.